assets/rich_text_components_definitions.ts

```diff
@@ -177,7 +177,7 @@ export = {
     "is_block_element": true,
     "customization_arg_specs": [{
       "name": "svg_filename",
-      "description": "The custom svg diagram filename",
+      "description": "The SVG Diagram Editor",
```

Comment:
This is a weird description for this customization arg... What does the customization arg specify? Is it the file name of the saved image? If so this is very weird. Why is the customization arg names "The SVG Diagram Editor"? Could you give me more context and I can maybe suggest a better name

Reply:
@nithusha21 This custom arg stores the filename of the saved svg file. But the problem is this description is displayed in the modal, thus if I use filename it looks weird in the modal. Thus I had to change it to fix it.

Reply:
What does it look like after? The screenshot in the description for this fix is not right...

Reply:
I am sorry but I am not able to follow you, the screenshot is correct right?

Reply:
Oh So the title in that image is controlled by this text... I see... That feels a bit weird, but I am fine with this change..